src/NuGet.Clients/NuGet.PackageManagement.UI/Models/PackageItemListViewModel.cs

```diff
@@ -22,11 +29,29 @@ namespace NuGet.PackageManagement.UI
     // Some of its properties, such as Latest Version, Status, are fetched on-demand in the background.
     public class PackageItemListViewModel : INotifyPropertyChanged
     {
-        private static readonly AsyncLazy<IEnumerable<VersionInfo>> LazyEmptyVersionInfo =
-            AsyncLazy.New(Enumerable.Empty<VersionInfo>());
+        private static readonly Common.AsyncLazy<IEnumerable<VersionInfo>> LazyEmptyVersionInfo =
+            Common.AsyncLazy.New(Enumerable.Empty<VersionInfo>());
 
-        private static readonly AsyncLazy<PackageDeprecationMetadata> LazyNullDeprecationMetadata =
-            AsyncLazy.New((PackageDeprecationMetadata)null);
+        private static readonly Common.AsyncLazy<PackageDeprecationMetadata> LazyNullDeprecationMetadata =
+            Common.AsyncLazy.New((PackageDeprecationMetadata)null);
+
+        private const int DecodePixelWidth = 32;
```

Comment:
Make it internal. That way, you can reference it on tests.

```suggestion
        internal const int DecodePixelWidth = 32;
```